requirements/base.txt

```diff
@@ -1,9 +1,10 @@
 interpolation>=2.2.3
 joblib>=1.2
+legacy-cgi; python_version >= '3.13'
```

Comment:
interpolation uses a python feature that has been long deprecated and now removed from python 3.13. This package adds legacy support.